src/lib/binaries/test.ts

```diff
@@ -14,6 +14,7 @@ const MAX_RPC_LIMIT = 100;
 const MAX_DOWNLOAD_LIMIT = 10 * 1024 * 1024;
 const MAX_UPLOAD_LIMIT = 5 * 1024 * 1024;
 const MAX_REQUEST_LIMIT = 110;
+const MAX_STORAGE_LIMIT = 1; // kb
```

Comment:
shouldn't this be 1024?

Reply:
updated in 2c7b85286d65254577b4c753ff89e931a874e1c3